contrib/coredns/policy/config.go

```diff
@@ -53,6 +55,13 @@ func policyParse(c *caddy.Controller) (*policyPlugin, error) {
 					return nil, err
 				}
 			}
+			if len(p.conf.ownIPEndpoint) > 0 && len(p.conf.debugSuffix) > 0 {
+				var buff bytes.Buffer
+				buff.Write([]byte(p.conf.ownIPEndpoint))
+				buff.Write([]byte("."))
+				buff.Write([]byte(p.conf.debugSuffix))
+				p.conf.ownIPEndpoint = buff.String()
```

Comment:
Should we use a separate param in conf to hold the ownIPEndpoint + debugSuffix? As of now there is no other use of ownIPEndpoint , so i used the same param itself.

Reply:
yes, that's ok. You can even require from user to provide FQDN, and not depend on debug suffix.
Please also update readme